### PR TITLE
add a11y and i18n text

### LINKED ISSUE #147: Add section on accessibility considerations

### PR DESCRIPTION
**Pull request by decentralgabe**
fixes #147 and #148

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/169.html" title="Last updated on Jun 22, 2023, 10:31 PM UTC (36870a9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/169/5b8a09d...36870a9.html" title="Last updated on Jun 22, 2023, 10:31 PM UTC (36870a9)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -832,11 +832,10 @@ <h2>Accessibility Considerations</h2>
...
          <p>
-            This section details the general accessibility considerations to take into
-            account when utilizing this data model.
-         </p>
-         <p class="issue" data-number="147">
-            Write accessibility considerations.
+            JSON Schemas are designed to be a machine-readable format which provides static
+            validation. As such, human readability is a secondary concern. When using a
+            <a>verifiable credential</a> to represent a schema, we recommend following the
+            guidance in the <a href="https://w3c.github.io/vc-data-model/#accessibility-considerations">VC Data Model</a>.
```
Comment (OR13) [inline index.html:838]: nit, use `data-cite` instead.

File: index.html [hunk 2]
```diff
@@ -850,11 +849,14 @@ <h2>Internationalization Considerations</h2>
...
+            JSON Schemas are JSON text intended primarily for machines to read, since they 
+            are used for strict static validation of data. Language and text direction concerns 
+            are addressed by the <a href="#json-schema-specifications">noted specification documents</a>
+            for JSON Schema itself.
          </p>
-         <p class="issue" data-number="148">
-            Write i18n considerations.
+         <p>
+            When using a <a>verifiable credential</a> to represent a schema, we recommend following the
+            guidance in the <a href="https://w3c.github.io/vc-data-model/#internationalization-considerations">VC Data Model</a>.
```
Comment (OR13) [inline index.html:859]: nit, use `data-cite` instead

**Review by OR13 — APPROVED** [review]

**Review by andresuribe87 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (decentralgabe) [reply index.html:838]: how can I use this?

File: index.html (hunk 2 above)
Comment (decentralgabe) [reply index.html]: updated

File: index.html (hunk 1 above)
Comment (decentralgabe) [reply index.html]: updated

**Pull request merged**